### PR TITLE
Pull in upstream improvements

### PR DESCRIPTION
**Pull request by felddy**
# 

## 🗣 Description

- GitHub Actions
- Fix code rot related to certbot container having old versions of boto3 libraries

## 💭 Motivation and Context

Keeping things up-to-date, and moving to new Docker org.

## 🧪 Testing

Ran all test suites.

## 🚥 Types of Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (causes existing functionality to change)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/entrypoint.sh [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 set -o nounset
 set -o errexit
+# shellcheck disable=SC2039
```
Comment (dav3r) [inline src/entrypoint.sh:7]: Don't we usually put in a comment explaining why a check is being disabled?

**Review by dav3r — APPROVED** [review]
Approved, with one comment

**Comment by dav3r** [comment]
Also @felddy, it looks like this repo only needs 1 reviewer for PRs- that should be changed to 2.

**Review by mcdonnnj — APPROVED** [review]
Looking good, but do see the comment by @dav3r

**Review by jsf9k — APPROVED** [review]
Approved, modulo the comment by @dav3r.

File: src/entrypoint.sh (hunk 1 above)
Comment (felddy) [reply src/entrypoint.sh:7]: Yes. I'll transfer commit comment to code: https://github.com/cisagov/certboto-docker/commit/8e9f90092dd5d98b8a8422e939d1c1e22ba0b9a0

Comment (felddy) [reply src/entrypoint.sh:7]: Comment added... good catch: https://github.com/cisagov/certboto-docker/pull/2/commits/db4220d1d138dfd0192a334ef7d6ee6f8812267c

Unfortunately I think my brain lets me get away with this when I write the commits with details.  Keep a look out!

**Review by felddy — COMMENTED** [review]
> Also @felddy, it looks like this repo only needs 1 reviewer for PRs- that should be changed to 2.

Now it needs 2!  TURN YOUR KEY SIR!  https://www.youtube.com/watch?v=rLMCjuge6oE

**Pull request merged**